distributed/tests/test_dask_collections.py

```diff
@@ -1,27 +1,19 @@
 from __future__ import annotations
 
 import pytest
-from packaging.version import parse as parse_version
 
 np = pytest.importorskip("numpy")
 pd = pytest.importorskip("pandas")
 
+from pandas.testing import assert_frame_equal, assert_index_equal, assert_series_equal
+
 import dask
 import dask.bag as db
 import dask.dataframe as dd
 
 from distributed.client import wait
 from distributed.utils_test import gen_cluster
 
-PANDAS_VERSION = parse_version(pd.__version__)
-PANDAS_GT_100 = PANDAS_VERSION >= parse_version("1.0.0")
-
-if PANDAS_GT_100:
```

Comment:
nit: I haven't found a declaration of the minimum version of `pandas` as `1.0.0` in `dask/distributed`, so I guess this is transitive through `dask/dask` (https://github.com/dask/dask/blob/main/setup.py#L17). Should we make this explicit in our own `requirements.txt`?

Reply:
From what I see, it's also used in the memory sampler, which should be independent of `dask[dataframe]` or am I mistaken?